app/containers/Sessions/containers/SettingsSessionPage/components/SessionSettings.js

```diff
@@ -173,6 +174,13 @@ const SessionSettings = ({
     );
   };
 
+  const updateSmsCode = (val, index) => {
+    const updatedAttributes = set(smsCodesAttributes, `${index}.smsCode`, val);
+    editSession({
+      smsCodesAttributes: updatedAttributes,
+    });
+  };
+
```

Comment:
I wonder if FE should be sending all of these attributes in PUT
<img width="547" alt="image" src="https://github.com/Michigan-State-University/cias-web/assets/86963976/36861475-1423-46f7-9aa7-a14f076973cc">